moviebot/database/db_users.py

```diff
@@ -0,0 +1,92 @@
+"""Class for managing the users database."""
+
+import sqlite3
+
+from werkzeug.security import check_password_hash, generate_password_hash
+
+from moviebot.database.db_manager import DatabaseManager
+
+
+class UserDB:
+    def __init__(self, users_db_path: str = "data/users.db") -> None:
+        """Class for managing the users database.
+
+        Args:
+            users_db_path: Path to users database.
+        """
+        self.manager = DatabaseManager(users_db_path)
+
+    def get_user_id(self, username: str) -> int:
```

Comment:
The method can return None, so I would update the return type as optional.
```suggestion
    def get_user_id(self, username: str) -> Optional[int]:
```

Reply:
Done